apps/saleor-app-checkout/mocks/consts.ts

```diff
@@ -10,6 +13,14 @@ export const testingVars = {
   adyenWebhookUsername: process.env.TEST_ADYEN_WEBHOOK_USERNAME ?? "",
 };
 
+if (IS_TEST) {
+  invariant(testingVars.mollieKey, "TEST_MOLLIE_KEY is not defined");
+  invariant(
+    testingVars.mollieProfileId,
+    "TEST_MOLLIE_PROFILE_ID is not defined"
+  );
```

Comment:
Why other values, such as `TEST_ADYEN_MERCHANT_ACCOUNT`, are not required?

Reply:
That't because we don't use them *yet* 🙈

Reply:
As we talked, let's try providing some kind of defaults which can be used just for replaying Pollyjs tests.

Reply:
I've provided default values in `.env` and also updated `.env.template`